packages/commands/src/index.ts

```diff
@@ -1250,10 +1250,8 @@ export namespace CommandRegistry {
   export function keystrokeForKeydownEvent(event: KeyboardEvent): string {
     let layout = getKeyboardLayout();
     let key = layout.keyForKeydownEvent(event);
-    if (!key || layout.isModifierKey(key)) {
```

Comment:
This would break our intended functionality as we need to handle modifier keys alone.
As we need to display an overlay element when the corresponding key Alt or Alt Shift is pressed.

For more context: [Add overlay UI element to display Jupyter tab bar keyboard shortcut number](https://github.com/jupyterlab/jupyterlab/issues/15090)

Reply:
Yes, I know the idea is to handle the modifier keys alone.
My comment was about a way to disable this functionality if not necessary, e.g. people who don't want to display the overlay. But this condition should probably be handled in jupyterlab.

Reply:
From what i can tell Jupiter does not currently have a way to detect if a user is using accessibility tooling, or which specific tools they need access to (e.g., keyboard only) making it out of reach to disable the functionality on a per user basis if not needed.

We could implement an opt-in option for users, but the challenge would be making these options discoverable. One way to do this would be to present the options to all users on app load.

I am happy to discuss this option further and possibly implement it in another PR.